breadcrumb/toc.yml

```
@@ -1,3 +1,11 @@
-- name: Docs
+- name: Learn
   tocHref: /
-  topicHref: /
\ No newline at end of file
+  topicHref: /
+  items:
+  - name: Partner Center REST API reference
+    tocHref: /partner-center/develop/partner-center-rest-api-reference/
+    topicHref: /partner-center/develop/partner-center-rest-api-reference
+    items:
+    - name: Azure Digital Twins
```

For the last item, you're pointing to technical content rather than developer content. I think here's where we would link in the nav for the current docset.
```suggestion
    - name: Azure Digital Twins REST API reference
```